acceptance/selftest/record_cloud/workspace-file-io/test.toml

```diff
@@ -0,0 +1,9 @@
+# hello, world\n base64 encoded in unix
+[[Repls]]
+Old = "aGVsbG8sIHdvcmxkCg=="
+New = "[HELLO-WORLD]"
+
+# hello, world\n base64 encoded in windows
+[[Repls]]
+Old = "aGVsbG8sIHdvcmxkDQo="
+New = "[HELLO-WORLD]"
```

Comment:
Where does this nuance show up?

I see the comment about newlines in the `.wsignore` file, but shouldn't the output always be verbatim?

Reply:
> but shouldn't the output always be verbatim?

The difference I believe is `\n`vs `\r\n` that's being printed on the console.

Reply:
I get that. What I wonder is, where does the `\n` or `\r\n` come from, if the `fs cat` command outputs file contents verbatim, and the input file doesn't have a newline.

Reply:
The input file here does have a newline. The one is volume-io file test does not, because the new line was hard to normalize in the API request payload.

Reply:
It's worth making this behave in a single way instead of accounting for the differences in the test itself.

We're likely seeing this: https://docs.github.com/en/get-started/git-basics/configuring-git-to-handle-line-endings

I suspect we can fix this by configuring everything under `acceptance` to use `eol=lf`.

Reply:
Good point, trying to address that in https://github.com/databricks/cli/pull/2865.